scripts/version_mapper.py

```diff
@@ -80,14 +88,38 @@ def update_switch_version_file(filename, new_version, cname, render_last):
 
     # Override the whole content of the version switches JSON file with the new
     # generated version data
-    with open(f"release/{filename}", "w") as switcher_file:
+    with open(f"release/{json_filename}", "w") as switcher_file:
         # Update JSON file with the new content
         json.dump(new_content, switcher_file, indent=4)
 
+    # Use the latest stable verion for formatting the announcement
+    with open(annoucement_filename, "r") as announcement_file:
+        content = announcement_file.read()
+        announcement_content = content.format(version=latest_stable_version)
+
+    # Include the announcement in all available release folders. Note that
+    # these are still accessible even if they are not included in the dropdown.
+    release_folders = [path for path in Path("./").iterdir() if path.is_dir()]
+    for release_folder in release_folders:
+        # Skip the latest stable version
+        if str(release_folder) == latest_stable_version:
+            continue
+
+        # Create an 'announcement.html' file within each one of the old versions
+        with open(f"{str(release_folder)}/announcement.html", "w") as file:
+            file.write(announcement_content)
+
 
 def parse_cli_arguments():
     """Parse all command line arguments."""
     parser = argparse.ArgumentParser(description="Version switcher JSON file updater.")
+    parser.add_argument(
+        "-a",
+        "--announcement_filename",
```

Comment:
```suggestion
        "--announcement-filename",
```

I am not much of a fan of underscores for CLI arguments

Reply:
This applies to the other arguments as well if you wish to change them - but totally up to you =). You can disregard my suggestion

Reply:
Yes, me neither... The issue is that, the name you place in here is the name for the variable and Python does not allow dash-separated variable names.

For example, the `--anouncement-filename` would be accessed with `args.announcement-file`, leading to an error.

Reply:
I see... the way you access the variable is the same as the CLI name you provide... Okay, no problem then 😄 I'll live with it, no biggie